aviary/subsystems/mass/flops_based/test/test_fuel_capacity.py

```diff
@@ -61,8 +61,8 @@ def configure(self):
             case_name,
             input_keys=[
                 Aircraft.Fuel.AUXILIARY_FUEL_CAPACITY,
-                Aircraft.Fuel.CAPACITY_FACTOR,
-                Aircraft.Fuel.DENSITY_RATIO,
+                Aircraft.Fuel.WING_FUEL_FRACTION,
+                Aircraft.Fuel.DENSITY,
```

Comment:
Are any of these tests running a check partials? You updated the core compute of fuel_capacity.py and that's typically an opportunity for partials mistakes to crop up.

Reply:
do_validation_test() auto-runs check_partials!